src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Rectangle implements Figure {
+    private String color;
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        this.color = color;
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq. units, length: " + length + " units, width: " + width + " units, color: " + color);
```

Comment:
Instead of concatenating strings in the 'draw()' method, consider using `String.format` for better readability and performance.